coalib/output/ConsoleInteraction.py

```diff
@@ -857,6 +858,19 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
+def check_deprecation(param_list):
+    """
+    Shows a deprecation warning message if the parameters
+    passed are not None.
+
+    :param param_list: This parameter contains a list of parameters
+                       being checked for deprecation.
+    """
+    for param_name, param_value in param_list.items():
+        if param_value is not None:
+            logging.warning('{} parameter is deprecated'.format(param_name))
```

Comment:
Could you move this to `misc`? To some utilities-module^^

Reply:
ok